governance-and-automation/security-compliance-checks/private-workspaces-shared-with-everyone.md

```diff
@@ -0,0 +1,43 @@
+---
+description: This article provides information on the Private Workspaces Shared with Everyone report.
+---
+
+# Private Workspaces Shared with Everyone
+
+Syskit Point detects private workspaces shared with everyone based on the settings defined in the tenant-wide [policy](../automated-workflows/private-workspaces-shared-with-everyone-admin.md). This helps you ensure your private workspaces are secure and only the people who should have access to the workspace have it. 
+
+The Private Workspaces Shared with Everyone is **a tenant-wide policy, and it cannot have task delegation enabled**. That means no tasks are created to resolve this policy, and no emails are sent to collaborators.
+
+The private workspaces shared with everyone **are still detected** as a part of the Security and Compliance checks feature. **This means that Syskit Point detects a vulnerability on a workspace** based on the applied policy, but it does not create tasks or send any emails to collaborators. 
+
+The purpose of this is to assist Syskit Point admins by
+bringing awareness of potential issues in their Microsoft 365 environment. 
+
+Private Workspaces Shared with Everyone are private workspaces **shared with a large number of users** through Everyone / Everyone except external users / All Users. 
+
+The policy can be applied to Microsoft Groups, Microsoft Teams, Sites, and OneDrive workspaces. 
```

Comment:
ne znam jel stavljamo microsoft groups inače u docsima, Microsoft 365 Groups je točnije, za sites bi isto stavio Communication sites. Isto tako kao i u prvom komentaru ovdje bi naveo Private Groups and Teams